stdlib/experimental/experimental.flux

```diff
@@ -1252,3 +1252,34 @@ builtin histogram : (
         bins: [float],
         ?normalize: bool,
     ) => stream[{T with _value: float, le: float}]
+
+// preview limits the number of rows and tables in the stream.
+// The group keys that are included are not deterministic and depends on the order
+// that the engine sends them.
+//
+// ## Parameters
+// - nrows: Maximum number of rows in each table.
+//
+// - ntables: Maximum number of tables returned by the transformation.
```

Comment:
```suggestion
// - ntables: Maximum number of tables to return.
```

Reply:
Does this parameter have a default value? The example below suggests that it does.